apps/web/src/server/lib/nunjucks-template-builders/summary-list-builder.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * @param {any} header
+ * @param {any[]} rowArray
+ * @returns {object}
+ */
+export function generateSummaryList(header, rowArray) {
+	const rows = [];
+	for (const row of rowArray) {
+		rows.push({
+			key: {
+				text: row.title
+			},
+			value: {
+				html: formatRowValue(row)
+			},
+			actions: {
+				items: [
+					{
+						href: row.actionLink,
+						text: row.actionText,
+						visuallyHiddenText: row.title
+					}
+				]
+			}
+		});
+	}
+	return {
+		card: {
+			title: {
+				text: header
+			}
+		},
+		rows: rows
+	};
+}
+
+/**
+ * @param {{ valueType: string; value: string; }} row
+ */
+function formatRowValue(row) {
```

Comment:
Minor, but I think you could just pass in the value here rather than the whole row